<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyApp</class>
 <widget class="QMainWindow" name="MyApp">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1075</width>
    <height>534</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Графики</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCustomPlot" name="gfplot" native="true">
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="comPanelFrame">
        <property name="maximumSize">
         <size>
          <width>237</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>7</number>
         </property>
         <property name="topMargin">
          <number>7</number>
         </property>
         <property name="rightMargin">
          <number>7</number>
         </property>
         <property name="bottomMargin">
          <number>7</number>
         </property>
         <item>
          <widget class="QFrame" name="ComStatusFrame">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>22</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="COMPortStatus">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>121</width>
              <height>22</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Com Port Status</string>
            </property>
           </widget>
           <widget class="QToolButton" name="actionResetCom">
            <property name="geometry">
             <rect>
              <x>200</x>
              <y>0</y>
              <width>20</width>
              <height>22</height>
             </rect>
            </property>
            <property name="text">
             <string>X</string>
            </property>
           </widget>
           <widget class="QToolButton" name="changeActivityCom">
            <property name="geometry">
             <rect>
              <x>170</x>
              <y>0</y>
              <width>22</width>
              <height>22</height>
             </rect>
            </property>
            <property name="text">
             <string>a</string>
            </property>
           </widget>
           <widget class="QToolButton" name="actionSelectCom">
            <property name="geometry">
             <rect>
              <x>200</x>
              <y>0</y>
              <width>22</width>
              <height>22</height>
             </rect>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Отображаемые данные</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="CheckBox_Voltage">
           <property name="text">
            <string>Напряжение</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="CheckBox_Current">
           <property name="text">
            <string>Сила тока</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="CheckBox_Power">
           <property name="text">
            <string>Мощность</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>21</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>21</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QCheckBox" name="lastDataCheckBox">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>216</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>Последние данные (cек)</string>
            </property>
           </widget>
           <widget class="QSpinBox" name="lastDataInput">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>170</x>
              <y>0</y>
              <width>51</width>
              <height>22</height>
             </rect>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="correctionMode">
             <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
            </property>
            <property name="showGroupSeparator" stdset="0">
             <bool>false</bool>
            </property>
            <property name="suffix">
             <string/>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
            <property name="stepType">
             <enum>QAbstractSpinBox::DefaultStepType</enum>
            </property>
            <property name="value">
             <number>10</number>
            </property>
            <property name="displayIntegerBase">
             <number>10</number>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="disableAutoRange">
           <property name="text">
            <string>Отключить прокрутку осей</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="AutoUpdateGraphCB">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Авто-обновление графика</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="updateComGraphButt">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Обновить график</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="openAnalitycsButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Статистика</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QFrame" name="comDataPortFrame">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>22</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="label_8">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>111</width>
              <height>22</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Данные COM Порта</string>
            </property>
           </widget>
           <widget class="QCheckBox" name="enablePortDataPlain">
            <property name="geometry">
             <rect>
              <x>120</x>
              <y>5</y>
              <width>16</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QPushButton" name="clearPortDataPlain">
            <property name="geometry">
             <rect>
              <x>160</x>
              <y>0</y>
              <width>61</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Очистить</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="COMPortDataPlain">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>200</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MyApp.qrc"/>
 </resources>
 <connections/>
</ui>
